doc/api/fs.md

```diff
@@ -434,10 +437,11 @@ changes:
                  buffers anymore.
 -->
 
-* `buffer` {Buffer|Uint8Array|string|Object}
+* `buffer` {Buffer|TypedArray|DataView|string|Object}
 * `offset` {integer} The start position from within `buffer` where the data
-  to write begins.
-* `length` {integer} The number of bytes from `buffer` to write.
+  to write begins. **Default:** `0`
+* `length` {integer} The number of bytes from `buffer` to write. **Default:**
+  `buffer.byteLength` or `buffer.length`
```

Comment:
Why isn't this just `buffer.byteLength`?

Reply:
I thought it would be what's used for the case where `buffer` is a string or an object, but actually it seems that this option is used as an encoding in those cases. I'll remove `buffer.length` here and open an issue about the string behavior.